Challenges.md

```diff
@@ -1,8 +1,8 @@
 ## Challenges
 <p><b>I'm great at many things but I struggle with...</b></p> 
-<p>Things that will break my focus in the classroom:</p>
-<p>Things that frustrate me in the classroom:</p>
-<p>Things that annoy me about group work:</p>
-<p>Things that keep me from getting my homework done:</p>
-<p>Situations that will make me angry:</p>
-<p>Situations that will make me shut-down:</p>
+<p>Things that will break my focus in the classroom: A class getting to boring and extremely quiet</p>
+<p>Things that frustrate me in the classroom:It getting to loud</p>
```

Comment:
I notice the dichotomy here of too quiet and too loud. Complete quiet actually really bothers me and I don't mind working in loud environments, so sometimes things will get too noisy before I take notice. Just let me know if the classroom gets too noisy and I'll quiet the noisy people down a notch =)